### PR TITLE
MWPW-140255: Merge handleOnNextClick and mobile progress indicator updates 

### PR DESCRIPTION
**Pull request by echampio-at-adobe**
- MWPW-139163: Small refactor to use selectedCards rather than a param in handleOnNextClick
- MWPW-138449: Shows mobile progress indicator at tablet widths
- Both MWPW-139163 and MWPW-138449 have been reviewed and tested with automation in the uar-integration branch. 

Resolves: 
[MWPW-140255](https://jira.corp.adobe.com/browse/MWPW-140255)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/drafts/quiz/quiz-2/?martech=off
- After: https://uar-integration--milo--adobecom.hlx.live/drafts/quiz/quiz-2/?martech=off

### CODE REVIEW

**Review by fullcolorcoder — APPROVED** [review]

**Review by colloyd — APPROVED** [review]

File: libs/blocks/quiz/quiz.css [hunk 1]
```diff
@@ -417,11 +409,19 @@ html[dir="rtl"] .quiz-step::after {
...
+  }
 }
 
 /* Desktop wide */
 @media screen and (min-width: 1200px) {
   .quiz-step-container.wide {
     width: 360px;
   }   
-}
+}
```
Comment (npeltier) [inline libs/blocks/quiz/quiz.css:427]: EOL is needed

File: libs/blocks/quiz/quiz.css [hunk 2]
```diff
@@ -417,11 +409,19 @@ html[dir="rtl"] .quiz-step::after {
   .quiz-option.has-icon .quiz-option-image {
     display: flex;
   }
+
```
Comment (npeltier) [inline libs/blocks/quiz/quiz.css:412]: is there a reason why those rules are being moved? i think changes like that should be avoided to make clearer PR and file histories ...

File: libs/blocks/quiz/quiz.css [hunk 3]
```diff
@@ -417,11 +409,19 @@ html[dir="rtl"] .quiz-step::after {
...
+  .quiz-step-container.top {
+    display: flex;
+  }
+  
+  .quiz-step-container.bottom{
```
Comment (overmyheadandbody) [inline libs/blocks/quiz/quiz.css:417]: Nit: please add a space between the selector and the curly brace, to match other styles. CSS Linter might complain about this too

**Review by auniverseaway — APPROVED** [review]

File: libs/blocks/quiz/quiz.css (hunk 2 above)
Comment (fullcolorcoder) [reply libs/blocks/quiz/quiz.css:412]: Thanks for taking the time to check this out. One of the tasks in this PR involved revamping breakpoints. It looks like some simple refactoring for that effort. I've opened another PR to resolve the concerns I can.
https://github.com/adobecom/milo/pull/1636

File: libs/blocks/quiz/quiz.css (hunk 3 above)
Comment (fullcolorcoder) [reply libs/blocks/quiz/quiz.css:417]: I've opened another PR to resolve this. https://github.com/adobecom/milo/pull/1636

**Review by mokimo — APPROVED** [review]

File: libs/blocks/quiz/quiz.css (hunk 2 above)
Comment (echampio-at-adobe) [reply libs/blocks/quiz/quiz.css:412]: Thanks, @npeltier, for the find, and @fullcolorcoder for the fix. It's has been merged into uar-integration

File: libs/blocks/quiz/quiz.css (hunk 3 above)
Comment (echampio-at-adobe) [reply libs/blocks/quiz/quiz.css]: Thanks, @overmyheadandbody, for the find, and @fullcolorcoder for the fix. It's has been merged into uar-integration

**Pull request merged**